odoo/tests/common.py

```diff
@@ -1345,15 +1345,15 @@ def clear(self):
                 registrations => Promise.all(registrations.map(r => r.unregister()))
             )
         """, 'awaitPromise': True})
+        # wait for the screenshot or whatever
+        wait(self._responses.values())
         self._logger.info('Deleting cookies and clearing local storage')
         self._websocket_request('Network.clearBrowserCache')
         self._websocket_request('Network.clearBrowserCookies')
         self._websocket_request('Runtime.evaluate', params={'expression': 'try {localStorage.clear();} catch(e) {}'})
         self.navigate_to('about:blank', wait_stop=True)
         # hopefully after navigating to about:blank there's no event left
         self._frames.clear()
-        # wait for the screenshot or whatever
-        wait(self._responses.values())
```

Comment:
I think this should remain here in order to properly wait for the various cleanup requests we've performed.

Reply:
Or we could specifically wait a second time at this place too.
On the other hand, I think that those cleanup requests are not very important as the browser will be either terminated or re-used for another test. Therefore it should have time to fulfill those simple requests.

Reply:
I agree. I changed accordingly.

Reply:
:heart: